CODEOWNERS

```diff
@@ -0,0 +1 @@
+* rtkwlf/rnd_arcticture
```

Comment:
```suggestion
*   @rtkwlf/rnd_architecture
```

I hate to be the bearer of bad news, but your team name is `rnd_architecture`

https://github.com/orgs/rtkwlf/teams/rnd_architecture